semcore/d3-chart/src/component/Chart/DistributionBarChart.type.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Intergalactic } from '@semcore/core';
+import { Flex } from '@semcore/flex-box';
+import { ScaleBand, ScaleLinear, ScaleTime } from 'd3-scale';
+import { BaseChartProps } from './AbstractChart.type';
+
+type BarKey = string;
+
+export type DistributionBarChartData = Array<Record<BarKey, string | number | Date>>;
```

Comment:
not sure about string... looks like we could work only with numbers (and maybe dates)